beps/0001-notifications-system/README.md

```diff
@@ -160,89 +168,202 @@ The following backend service interfaces are added as part of this proposal.
 ```ts
 // TODO - We may want to add an additional wrapping here with interfaces for Notification and NotificationParameters
 
-interface SendNotificationRequest {
-  receivers: SignalReceivers;
+export type NotificationRecipients =
+  | {
+      type: 'entity';
+      entityRef: string | string[]; // typically a user or group entity, otherwise `spec.owner` of the entity/entities
+    }
+  | {
+      type: 'broadcast'; // all users
+    };
 
-  /* TODO - please contribute :) */
-}
+export type NotificationType = 'undone' | 'done' | 'saved';
 
-interface SendNotificationResponse {
+export type Notification = {
```

Comment:
Would it make sense to have `payload` field to break this flat structure?

Reply:
Don't know, I think it makes it a bit more confusing to handle in the UI side. Let's discuss on discord!

Reply:
I'm also thinking a bit grouping of fields could be good here